app/upgrades/v4/sequencer_params.go

```diff
@@ -0,0 +1,15 @@
+package v4
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sequencerkeeper "github.com/dymensionxyz/dymension/v3/x/sequencer/keeper"
+	sequencertypes "github.com/dymensionxyz/dymension/v3/x/sequencer/types"
+)
+
+func migrateSequencerParams(ctx sdk.Context, k *sequencerkeeper.Keeper) {
+	// overwrite params for rollapp module due to proto change
+	// (Note: all of them have changed, including min bond)
```

Comment:
what's about min bond? is it okay if we overwrite it with the default value?

Reply:
yes because min bond changed to 100